homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,24 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    int a = 0;      //wyraz n-2
+    int b = 1;      //wyraz n-1
+    int sum = 0;
+    if (sequence == 1 || sequence == 2) return 1;
+    else {
+        for (int i = 0; i < sequence - 1; ++i) {
+        
+            sum = a + b;
+            b += a;
+            a = b - a;
+        }
+        return sum;
+    }
```

Comment:
Add space after `if`, `for`, `while` etc...

When you use return inside `if` you don't need `if else` or `else`. Just write `if (sth) { return blabla; }`

One command one line